src/index.html

```diff
@@ -12,11 +12,54 @@
     <title>Aurator</title>
 </head>
 <body class="app">
-    <img src="img/desktop/naprawa__foto.png" alt="foto">
     <div class="container">
-        <div class="row">
-            <div class="col-sm-12">
-                <h2>Aurator</h2>
+        <div class="top">
+            <div class="nav">
+                <div class="nav-logo">
+                    <img src="img/nav__logo.png" alt="logo">
+                </div>
+                <div class="nav-menu">
+                    <ul class="nav-menu__list d-none d-lg-inline-block">
+                        <li class="d-inline"><a class="nav-link h-txt9" href="#">NAPRAWA I REGENERACJA</a></li>
+                        <li class="d-inline"><a class="nav-link h-txt9" href="#">PRODUKCJA</a></li>
+                        <li class="d-inline"><a class="nav-link h-txt9" href="#">CENTRUM NAPRAW AWARYJNYCH LOKOMOTYW</a></li>
```

Comment:
I am not big fun of `h-*`. Maybe add definition of `h-txt9` to `nav-link`.

```html
<a class="nav-link" href="#">Produkcja</a>
<a class="nav-link nav-link--orange" href="#">Kontakt</a>
<a class="nav-link nav-link--mobile" href="#">Produkcja</a>
<a class="nav-link nav-link--orange nav-link--mobile" href="tel:+48323012097">+32 301 20 97</a>
```

Reply:
You mean with the use `@extend` in scss? Or to avoid the `h-txt9` class at all and just add its the styling to all the classes that needs it? This way it won't be reusable and more difficult to mantain, which I think are one of the main reasons for using BEM.

Reply:
We could use extend or we could use variables for fontsize and colors. We should talk how we understand BEM, cause I see some discrepancy.